CHANGELOG.md

```diff
@@ -5,11 +5,13 @@
     - TXRMDataReader is deprecated in favour of ZEISSDataReader 
 
 * 21.4.1
-  - Removed prints from unittests and cleanup of unittest code.
-  - CMake: 
-    - install script re-allows selection of non default install directory ([#1246](https://github.com/TomographicImaging/CIL/issues/1246))
-  - TIFF writer uses logging
-  - Added unittests for TIFF functionality
+
+ - Add ndim property for DataContainer class.
```

Comment:
This should be moved to the 22.0.0 section of the change log as 21.4.1 has been released.
Indenting needs correcting back to how it was under 21.4.1